plugins/redshift/dbt/include/redshift/sample_profiles.yml

```diff
@@ -0,0 +1,25 @@
+default:
```

Comment:
Let's leave `host: [host]` and `dbname: [dbname]` generic, since both can be different depending on the cluster / connection